src/components/article-page.js

```diff
@@ -19,6 +20,14 @@ export default class Article extends PureComponent {
       <div>
         <LeadingBlock />
         <Body content={_.get(post, 'content.api_data')} />
+        <Metadata
+          categories={post.categories}
+          designers={post.designers}
+          photographers={post.photographers}
+          tags={post.tags}
+          themeColor="#fb9cec"
+          writers={post.writters}
```

Comment:
[TODO] We need to add `engineers` and `extendLine` if the components are used for general purpose

Reply:
Sure,  will add `engineers` first.
And will add `extend_byline` after mockup ready.